src/components/ActionPage/MiniBackoffice.tsx

```diff
@@ -108,14 +112,11 @@ export const MiniBackoffice = () => {
     <Card title="Actions performed by Fractal's servers" width="90%">
       <CardBodyContainer>
         <Text>
-          These transaction show you how Fractal updates the DID Registry. The
-          user would not be asked to do these transactions.{" "}
+          This window shows you how Fractal updates the DID Registry. The user
+          would not be asked to do these transactions but you can perform them
+          for this demo.{" "}
```

Comment:
Are you sure we need the `{" "}` here? Doesn't seem to be the case.

Since we're at it, please check the same thing for the line below.

Reply:
prettier inserted these.